flake8_trio/visitors/visitor100.py

```diff
@@ -7,9 +7,8 @@
 """
 from __future__ import annotations
 
-from typing import Any
+from typing import TYPE_CHECKING, Any
 
-import libcst as cst
```

Comment:
Aaaand, let's disable this as well - it's pretty verbose and doesn't actually have any performance benefits here at all; we're executing all the imports already when we import the matchers.

(perils of starting with such a long list of available lint rules!)

Reply:
Huh, looks like flake8-type-checking has explicit logic about ignoring the case when you import a submodule. I'll add `libcst` to https://beta.ruff.rs/docs/settings/#exempt-modules for now but might want to stick with `flake8-type-checking` over ruff and/or open an issue in ruff.

Reply:
opened https://github.com/charliermarsh/ruff/issues/4667